lazy_tensor_core/lazy_tensor_core/csrc/ts_backend/aten_ltc_ts_type.cpp

```diff
@@ -376,7 +386,7 @@ at::Tensor LazyNativeFunctions::empty(
                                   .pinned_memory(pin_memory)
                                   .dtype(dtype);
   auto x_result = at::empty(size, options, memory_format);
-  return CreateLtcTensor(x_result, bridge::GetLtcDevice(device));
+  return CreateLtcTensor(x_result, GetLtcDevice(device));
```

Comment:
I guess it's still being called, but, i'm not sure what the reason is

Reply:
Yea, it's more like a helper to do some checks before the actual conversion.